packages/framework/src/Model/AdvancedSearchOrder/Filter/OrderCustomerIdFilter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\AdvancedSearchOrder\Filter;
+
+use Doctrine\ORM\QueryBuilder;
+use Shopsys\FrameworkBundle\Component\FlashMessage\FlashMessage;
+use Shopsys\FrameworkBundle\Model\AdvancedSearch\AdvancedSearchFilterInterface;
+use Shopsys\FrameworkBundle\Model\Customer\CustomerFacade;
+use Shopsys\FrameworkBundle\Model\Customer\Exception\CustomerNotFoundException;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+
+class OrderCustomerIdFilter implements AdvancedSearchFilterInterface
+{
+    public const string NAME = 'customerId';
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\CustomerFacade $customerFacade
+     * @param \Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface $flashBag
+     */
+    public function __construct(
+        protected readonly CustomerFacade $customerFacade,
+        protected readonly FlashBagInterface $flashBag,
+    ) {
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return self::NAME;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getAllowedOperators(): array
+    {
+        return [
+            self::OPERATOR_IS,
+            self::OPERATOR_NOT_SET,
```

Comment:
:thinking: 

Translation of this operator may be a little misleading ? It says "not entered", will administrator know what does it mean or filter ? 
I believe "not registered customer" would fit better. I know it is general translation, but needed to point it out